src/sinks/elasticsearch/integration_tests.rs

```diff
@@ -293,7 +293,7 @@ async fn auto_version_aws() {
             },
         )),
         endpoints: vec![aws_server()],
-        aws: Some(RegionOrEndpoint::with_region(String::from("localstack"))),
+        aws: Some(RegionOrEndpoint::with_region(String::from("us-east-1"))),
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❓ question:** ‏were these intended to be reverted?

Reply:
Yes. I updated the version of localstack we are running the integration tests against. Newer versions are multi region aware and no longer have a `localstack` region. The default is now `us-east-1`.